app/styles/gavant-ember-table/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$table-hover-bg-opaque: #ececec !default;
+$primary: #285a92 !default;
+$white: #fff !default;
+$table-border-color: rgba(#000, 0.075) !default;
+$font-family-base: 'Lato', sans-serif !default;
+$text-muted: lighten(#6c757d, 15%);
```

Comment:
i would prefix all your addon sass vars with like `$gavant-table-` to avoid naming collisions with the project's/other addon's variables.